### PR TITLE
Reuse conditionID when we add a realit.io question

### LINKED ISSUE #363: Reuse conditionID when we add a realit.io question
Check if the question was already used to prepare a condition before. If it was, reuse that conditionId (skipping the second step).

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #363

### CODE REVIEW

File: app/src/services/cpk.ts [hunk 1]
```diff
@@ -182,18 +182,22 @@ class CPKService {
...
+          logger.log(
+            `ConditionId found: '${conditionId}', Outcome slot count: '${outcomeSlotCount}'`,
+          )
         } catch (err) {
           logger.log(`ConditionId is not already prepared: '${err.message}'`)
         }
       }
 
-      if (!conditionLog) {
+      if (outcomeSlotCount.isZero()) {
```
Comment (fvictorio) [inline app/src/services/cpk.ts]: Maybe we can hide this complexity behind the service and just have a method `conditionExists` or something like that.

File: app/src/services/conditional_token.ts [hunk 2]
```diff
@@ -178,6 +193,40 @@ class ConditionalTokenService {
 
     return conditionId
   }
+
+  getConditionIdFromLogs = async (conditionId: string): Promise<ConditionLog> => {
```
Comment (mariano-aguero) [inline app/src/services/conditional_token.ts]: This function is not necessary, but I think it does not affect if we keep it, and maybe could be usefull

Comment (fvictorio) [reply app/src/services/conditional_token.ts]: We either keep this function and remove `getQuestionId`, or we remove this one. We can't keep both of them, they are almost equal.

File: app/src/services/cpk.ts (hunk 1 above)
Comment (mariano-aguero) [reply app/src/services/cpk.ts]: Done!

File: app/src/services/conditional_token.ts (hunk 2 above)
Comment (mariano-aguero) [reply app/src/services/conditional_token.ts]: Remove it

File: app/src/services/conditional_token.ts [hunk 3]
```diff
@@ -194,38 +193,9 @@ class ConditionalTokenService {
...
-
-    const { oracle, questionId, outcomeSlotCount } = event.values
-
-    return {
-      conditionId,
-      oracle,
-      questionId,
-      outcomeSlotCount,
-    }
+  isConditionExists = async (conditionId: string): Promise<boolean> => {
```
Comment (fvictorio) [inline app/src/services/conditional_token.ts]: Either `conditionExists` or `doesConditionExist`.

Comment (mariano-aguero) [reply app/src/services/conditional_token.ts]: Done!

**Pull request merged**